docs/INTEGRATION.md

```diff
@@ -198,3 +202,50 @@ You should now be able to run the Cypress Test Runner by running:
 ```shell
 yarn cypress-open
 ```
+
+# Integration testing with Playwright
+
+Like Cypress, [Playwright](https://playwright.dev/) is also an end-to-end testing framework that uses a browser. Playwright uses the Chrome DevTools Protocol to execute actions in the browser, unlike Cypress which injects code directly into the browser's execution loop. The result of this is that Playwright tests more accurately reflect an apps behaviour in the browser, since it's using standardised APIs and does not need to modify the browser to run its tests.
+
+## Setup
+
+_Note: This was tested on Ubuntu 24.04_
+
+First, you'll need to install Playwright and its dependencies. Assuming you have NodeJS and Yarn installed (see [HACKING.md](./HACKING.md#setup-maas-ui-node-and-yarn)), run the following command:
+
+```sh
+yarn playwright install --with-deps
+```
+
+You might need to restart your system after running this command. You'll also need to make sure you have a MAAS instance up and running - take note of the URL.
+
+## Running tests
+
+**Playwright expects a MAAS instance to be running at `http://0.0.0.0:5240` to test against.** If you've already got a local MAAS instance, then you should be good to go. If you followed the [LXD backend guide](./LXD.md) or have your MAAS running on another system, you'll need to change a few things before your tests will run.
+
+1. In `[playwright.config.ts](../playwright.config.ts)`, change `baseUrl` to your MAAS URL.
```

Comment:
Why `../`?

Reply:
The file is one directory up from where this doc lives